src/parts/InterlayHelmet/index.tsx

```diff
@@ -36,6 +37,7 @@ const InterlayHelmet = (): JSX.Element => (
       property='twitter:image'
       content={`${APP_DOMAIN}/${PUBLIC_ASSETS_FOLDER_NAME}/${OPEN_GRAPH_IMAGE_FILE_NAME}`}
     />
+    {ANALYTICS_CODE && <script src='https://cdn.usefathom.com/script.js' data-site={ANALYTICS_CODE} defer></script>}
```

Comment:
Conditional check to avoid tag rendering when application is running locally (i.e. when there is no value for `ANALYTICS_CODE`)